fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -1383,7 +1395,8 @@ void removeLog(Reference<LogGenerationData> logData) {
 ACTOR Future<Void> tLogCore(
     Reference<TLogServerData> self,
     std::shared_ptr<std::unordered_map<TLogGroupID, Reference<LogGenerationData>>> activeGeneration,
-    TLogInterface_PassivelyPull tli) {
+    TLogInterface_PassivelyPull tli,
+    UID epochId) {
```

Comment:
`epoch` means generation, which is of `Version` type. From the context, better call this `recruitmentId` or `recruitId` for short.